app/controllers/spree/taxons_controller_decorator.rb

```diff
@@ -1,4 +1,5 @@
 Spree::TaxonsController.class_eval do
+
   def show
     if stale? [@taxon, @products, @taxonomies, simple_current_order]
```

Comment:
In default spree, this line is something like this:
<img width="1035" alt="Screen Shot 2021-12-07 at 12 34 19 AM" src="https://user-images.githubusercontent.com/78048076/144910166-20eb2ccc-30c2-4b06-8c06-755cc0d403b0.png">
Please confirm that, we copied the code from spree 4-3-stable

Reply:
its change by duke same as per bestflavour

Reply:
@Umer, I think you discussed this point with  me and we agreed that the Best case will not apply here. As BEST version was spree-3.7 and natures version is 4-3. There is lot of difference.  After detailed discussion we made the same mistake. 

Please justify this point. I need solid justification in this.

Reply:
@Umer, the  feedback is still not address.  Please check again